### PR TITLE
Feature/multi webserver support

### PR DESCRIPTION
**Pull request by davidporter-id-au**

### CODE REVIEW

**Comment by davidporter-id-au** [comment]
Urgh, Apologies for the notifications. I've managed to conflict somehow. I'll sort that out first.

File: src/verify.js [hunk 1]
```diff
@@ -80,43 +104,35 @@ module.exports = (function() {
...
+                url: "http://localhost:3000" + path,
+                headers: interaction.request.headers ? interaction.request.headers : {},
+                body: interaction.request.body,
+                method: interaction.request.method,
+                json: true
+            };
+
+            request(options, function(err, res, body){
+                if(err){
+                    console.error('Error in making GET request', err);
```
Comment (davidduffett) [inline src/verify.js]: I think this message needs to be updated to take into account the method, rather than hardcoded "GET"

File: src/verify.js [hunk 2]
```diff
@@ -80,43 +104,35 @@ module.exports = (function() {
...
+                    done(err);
+                }
+                done(verifier.verify(interaction, res));
             });
-          } catch(err) {
+        }
+        catch(err) {
+            console.log('caught error in sending GET request: ', err.stack);
```
Comment (davidduffett) [inline src/verify.js]: Same with this one - hardcoded "GET"

File: src/verify.js [hunk 3]
```diff
@@ -80,43 +104,35 @@ module.exports = (function() {
...
+        try {
+            var path = (interaction.request.query === undefined) ? interaction.request.path :
+                    interaction.request.path + "?" + interaction.request.query;
+
+            var options = {
+                url: "http://localhost:3000" + path,
+                headers: interaction.request.headers ? interaction.request.headers : {},
+                body: interaction.request.body,
+                method: interaction.request.method,
+                json: true
```
Comment (davidduffett) [inline src/verify.js]: The request might not necessarily be json, but we can handle this in the future.

File: src/verify.js (hunk 2 above)
Comment (davidporter-id-au) [inline src/verify.js]: Whoops, that was stupid

**Pull request closed without merging**